src/repositories/SlotRepository.ts

```diff
@@ -1,12 +1,37 @@
-import { Repository } from "typeorm";
 import Database from "../providers/Database";
 import Slot from "../entities/Slot";
+import ISlotRepository from "../types/Slot/ISlotRepository";
 
-const slotRepository = (): Repository<Slot> => Database.getConnection()
+const slotRepository = (): ISlotRepository => Database.getConnection()
   .getRepository(Slot).extend({
+    disableSlotsByRoomId(roomId: number): Promise<void> {
+      return this.createQueryBuilder()
```

Comment:
We must try to be consistent in how we implement query logic. Otherwise, it gets really confusing to look for an implementation reference, we are more error-prone, it becomes harder to understand why we do stuff differently in some places, etc... Imagine onboarding a new developer and finding a query builder in one repo for delete, and then a simple `repo.remove` in another one.

We either use _query builder_ everywhere as default, or we leave it for exceptional cases and then we use repository basic methods such as `remove`, `find`, `create`, and so on. Not sure if you decided to use this because there was a compelling reason to do so or if you were just doing trial and error and you got this one working, but we need to either change it to use `repo.remove` or update `findAllSlotsByRoomId` to also use query builder syntax please.

Reply:
I decided to change findAllSlotsByRoomId to query Builder syntax. I didn't find how to do the update query for many items without the query builder.